### PR TITLE
#154877085 prominent product quantity

### PR DESCRIPTION
**Pull request by daddychukz**
### What does this PR do?
it makes the product quantity input field more visible
it reposition the Tour Icon on the Navigation Bar

### Description of Task to be completed?
* Increase and reposition the product quantity input field
* Reposition Tour button on Nav Bar

### How should this be manually tested?
```
Type reaction in your terminal to run the application
Navigate to localhost:3000 in your browser
Locate the Tour button on the Nav Bar
Click on any product
Locate the product quantity input field above Add to Cart button
```
Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
```
story type: feature
story Id: 154877085
story title: Make product quantity more Visible
```

Screenshots (if appropriate)
screen shot 2018-02-02 at 4 22 05 pm

### CODE REVIEW

File: imports/plugins/included/default-theme/client/styles/tagNav.less [hunk 1]
```diff
@@ -394,12 +394,12 @@
 
 // Tour=======================
 .tourBtn {
-  height: 40px;
   width: 100px;
   background: #256188;
-  position: absolute;
+  // position: absolute;
```
Comment (sachungo) [inline imports/plugins/included/default-theme/client/styles/tagNav.less]: Delete

File: imports/plugins/included/default-theme/client/styles/tagNav.less [hunk 2]
```diff
@@ -413,8 +413,8 @@
   font-weight: bold;
   border: none;
   color: white !important;
-  position: absolute;
-  top: 30%;
+  // position: absolute;
```
Comment (sachungo) [inline imports/plugins/included/default-theme/client/styles/tagNav.less]: Delete

File: imports/plugins/included/default-theme/client/styles/tagNav.less [hunk 3]
```diff
@@ -413,8 +413,8 @@
...
+  // top: 30%;
```
Comment (sachungo) [inline imports/plugins/included/default-theme/client/styles/tagNav.less]: Delete

File: imports/plugins/included/payments-wallet/server/methods/wallet.js [hunk 4]
```diff
@@ -10,13 +11,35 @@ Meteor.methods({
    * @return {String} ownerEmail
    */
   "wallet/create-user-wallet": function (ownerEmail) {
+    console.log("createeeeee", ownerEmail);
     check(ownerEmail, String);
 
     Wallets.insert({ ownerEmail });
   },
+  // /**
```
Comment (sachungo) [inline imports/plugins/included/payments-wallet/server/methods/wallet.js]: Delete line 19 to 38 if not being used

File: imports/plugins/included/payments-wallet/server/methods/wallet.js [hunk 5]
```diff
@@ -10,13 +11,35 @@ Meteor.methods({
...
+  //   const results = {
+  //     saved: true,
+  //     response: response
+  //   };
+  //   return results;
+  // },
 
   "wallet/get-user-walletId": function (ownerEmail) {
     check(ownerEmail, String);
+    // console.log(ownerEmail);
```
Comment (sachungo) [inline imports/plugins/included/payments-wallet/server/methods/wallet.js]: Delete

**Review by sachungo — CHANGES_REQUESTED** [review]
As this PR contains changes from other PRs, which I have already reviewed, you'll have to merge staging into this branch once the other PRs have been merged.

Meanwhile, I have left minor feedback to be addressed.

File: imports/plugins/included/ui-search/lib/components/searchModal.js [hunk 6]
```diff
@@ -1,6 +1,7 @@
 import React, { Component } from "react";
```
Comment (sachungo) [inline imports/plugins/included/ui-search/lib/components/searchModal.js]: If you haven't made any change to this file, can you revert back to what is on staging branch? That is copy and paste the content here.  There are quality issues in this file, which were fixed in one of the PRs merged to staging. Example is the difference in [this lines](https://github.com/andela/annie-hall-rc/blob/staging/imports/plugins/included/ui-search/lib/components/searchModal.js#L11-L47)

Comment (daddychukz) [reply imports/plugins/included/ui-search/lib/components/searchModal.js]: Fixed!

File: imports/plugins/included/ui-search/lib/components/searchModal.js [hunk 7]
```diff
@@ -145,6 +145,132 @@ class SearchModal extends Component {
     );
   }
 
+  constructor(props) {
```
Comment (sachungo) [inline imports/plugins/included/ui-search/lib/components/searchModal.js]: Lines 148 to 273 repeats already defined methods in the class. Please delete them and include any change you might have added to the methods in the existing class methods. This [gives a better view of the repetition](https://github.com/andela/annie-hall-rc/blob/0589238e256f0206a61ef84e7788b98ccf31cc4a/imports/plugins/included/ui-search/lib/components/searchModal.js#L148-L272). Those methods exist in line 146 backwards

Comment (daddychukz) [reply imports/plugins/included/ui-search/lib/components/searchModal.js]: fixed

**Review by sachungo — APPROVED** [review]

**Pull request merged**